pages/common/yesod.md

```diff
@@ -0,0 +1,24 @@
+# yesod
+
+> A framework to develop Haskell web applications.
+> All yesod commands are invoked through the stack project manager.
+
+- Create a new scaffolded site:
+
+`stack new my-project yesod-sqlite`
+
+- Install the Yesod CLI tool within a Yesod scaffolded site:
+
+`stack build yesod-bin cabal-install --install-ghc`
+
+- Start development server:
+
+`stack -- exec yesod devel`
+
+- Touch files with altered Template Haskell dependencies:
+
+`stack -- exec yesod devel`
```

Comment:
Both this and the previous example use the same command. Ideally we would use the same description if the command is the same. Can we do anything to combine these two ? Or maybe have different commands for each ?